3d/flap.scad

```diff
@@ -39,8 +39,18 @@ letter_facet_number = 100;
 vertical_keepout_mode = 1;              // 0=ignore; 1=highlight; 2=cut
 vertical_keepout_size_factor = 1.1;     // Expand calculated keepout region by this factor. 1=no expansion, 1.5=50% expansion, etc
 
+// generate lower half of the flap
+generate_second_half = true;
```

Comment:
Is this needed? I noticed it's set to false in the new script, and I didn't quite understand it's usage in this file (since it adds a second `flap_with_letters`, but `flap_with_letters` should already be double-sided, I think?)